DESCRIPTION

```diff
@@ -2,27 +2,30 @@ Package: etn
 Title: Access and Process Data from the European Tracking Network
 Version: 0.1.0
 Authors@R: c(
-    person("Stijn", "Van Hoey", email = "stijn.vanhoey@inbo.be", role = c("aut", "cre")),
-    person("Damiano", "Oldoni", email = "damiano.oldoni@inbo.be", role = "aut"),
-    person("Peter", "Desmet", email = "peter.desmet@inbo.be", role = "aut"))
-Description: Package with functions to access and process data from the European Tracking Network hosted by VLIZ.
-Depends: R (>= 3.4.2)
+    person("Stijn", "Van Hoey", email = "stijn.vanhoey@inbo.be", role = c("aut", "cre"), comment = c(ORCID = "0000-0001-6413-3185")),
+    person("Damiano", "Oldoni", email = "damiano.oldoni@inbo.be", role = "aut", comment = c(ORCID = "0000-0003-3445-7562")),
+    person("Peter", "Desmet", email = "peter.desmet@inbo.be", role = "ctb", comment = c(ORCID = "0000-0002-8442-8025"))
+    )
+Description: Package with functions to access and process data from the European
```

Comment:
As I bumped into this [prepare-for-cran](https://github.com/ThinkR-open/prepare-for-cran/blob/master/README.md#do-not-put-package-in-the-description) overview this week, I found out: "Neither the title nor the description should start with "A package..." or with the name of the package." 
(we should take this into account for waterinfo/... as well ;-)

Reply:
Won't adapt this now, but good to know for the future!